tensorflow/python/ops/array_ops.py

```diff
@@ -259,11 +259,8 @@ def rank_internal(input, name=None, optimize=True):
       return gen_array_ops.rank(input, name=name)
 
 
-# DEPRECATED use init_ops.zeros_initializer
-# TODO(irving) Move it to init_ops.py
-def zeros_initializer(shape, dtype=dtypes.float32, partition_info=None):
-  """An adaptor for zeros() to match the Initializer spec."""
-  return zeros(shape, dtype)
+#DEPRECATED
+zeros_initializer = init_ops.zeros_initializer
```

Comment:
Never mind -- this will not work, you'd have to import init_ops, which you can't. Sorry. Rever this, it was a bad idea.